nemo/collections/asr/models/label_models.py

```diff
@@ -380,12 +385,22 @@ def setup_finetune_model(self, model_config: DictConfig):
 
     @torch.no_grad()
     def get_embedding(self, path2audio_file):
+        """
+        returns the speaker embeddings for a provided audio file
```

Comment:
returns -> Return (Capitalize + If you want to unify the verb form with other docstrings)